### ISSUE TITLE
libexecinfo: checksum error

### ISSUE THREAD

**Issue by X547** [haikuports/haikuports#9545]
```
Saving to: ‘/Haiku/data/packages/haikuports/sys-libs/libexecinfo/download/libexecinfo-1.1.tar.bz2’
     0K                                                      100% 1.93M=0.001s
2023-09-30 20:27:07 (1.93 MB/s) - ‘/Haiku/data/packages/haikuports/sys-libs/libexecinfo/download/libexecinfo-1.1.tar.bz2’ saved [1097/1097]
Validating checksum of libexecinfo-1.1.tar.bz2
Error: Expected SHA-256: c9a21913e7fdac8ef6b33250b167aa1fc0a7b8a175145e26913a4c19d8a59b1f
Error: Found SHA-256:    a78edec6413703670578c945a6d4782bcc3728a334427cfd6b62a7055ce6890b
```

**Comment by Begasus** [comment]
Could you check? https://github.com/haikuports/haikuports/pull/9549/files

**Comment by korli** [comment]
merged in d43f375edcc823b14bb70da0c16ed7a6cba0693d
